dialoguekit/core/domain.py

```diff
@@ -10,20 +10,30 @@ class Domain:
     def __init__(self, config_file: str) -> None:
         """Represents domain knowledge.
 
+        The YAML configuration file should contain at least the fields name and
+        slot_names.
```

Comment:
Should we check for these fields and throw an exception if they're not present?

Reply:
Good point! I added a check for the required fields.